R/zzz.R

```diff
@@ -6,27 +6,14 @@
 }
 
 # This package has specific methods for the `tunable()` and `required_pkgs()`
-# generic. That generic is defined in the `tune` package. As of R 4.0, we need to register them.
+# generic. That generic is defined in the `generics` package. As of R 4.0, we need to register them.
```

Comment:
Can remove outdated comments

Reply:
A question for you... is there something different about the `tidy` methods for steps and checks that we _do_ need to lazily register them, even though we are reexporting `tidy` from generics also? For example, see lines 15-24 or so.

Reply:
Oh interesting. Probably not. You can probably remove the lazy registration of those too. They may have been from before we had generics, and had to Suggest broom and lazy register methods there